aws.yaml

```diff
@@ -1,4 +1,101 @@
 - active: false
+  apps:
+    - app: chart-operator
+      version: 0.11.2
+      use-upgrade-force: true
+      changelogs:
```

Comment:
Not very happy with this - we need to figure out where changelogs are supposed to live. This will blow up these yamls massively.

Reply:
> we need to figure out where changelogs are supposed to live

@MarcelMue Yes I think it needs to be this repo but that we need to find a better home for the changelogs.

Reply:
@MarcelMue @rossf7 How about keeping release tracks on CHANGELOG.md for every installation type? Just as what we do in normal apps repo.

Reply:
But how do we link the apps here with the changelogs in app repos?

We need to keep the versions and changelogs exposed by the version endpoint in cluster-operator.

```
$ kg port-forward cluster-operator-2-0-1-5486684645-mwv8x 8000:8000
$ curl http://localhost:8000/ | jq .
```

What about using this structure but in a separate `/changelogs/aws.yaml` file?

Reply:
@rossf7 Only changelogs you mean by that, right?

Reply:
Yes just the changelogs so this file is cleaner. But I think you'll need the app and version in both files so the data can be merged.

Reply:
One thing that I don't like about a separate `/changelog/` files; it's not clear which changelogs related to which component version. That would be too trivial. Someone needs to write a new `release-note` by summarising all changelogs on it. 
At the moment, I can't think which way is better, any ideas?

Reply:
> But I think you'll need the app and version in both files so the data can be merged.

Yes, with this information, someone can merge them and write new `release-note` if only we keep changelogs as latest always.

Reply:
The release notes are written for humans whereas these changelog yamls will be for machines. So there will be some differences. But the yaml files should help with writing the release notes.

I think we will need a full history of the changelogs so yes the files will become big. We could have a file per release but AIUI the release yamls are mounted in a configmap so they can be accessed by cluster-operator.

What about updating the PR with the new structure so we can see what it looks like?